concordia/models.py

```diff
@@ -66,6 +68,41 @@ class Collection(models.Model):
     def __str__(self):
         return self.title
 
+    def copy_images_to_collection(self, url, collection_path):
+        result2 = None
+        try:
+            result = download_async_collection.delay(url)
+            result.ready()
+            result.get()
+            result2 = check_completeness.delay()
+            result2.ready()
+            result2.get()
+        except Exception as e:
+            print(e)
+            pass
+
+        if result2 and not result2.state == 'PENDING':
+            if os.path.isdir(collection_path):
+                shutil.rmtree(collection_path)
+            shutil.copytree('/concordia_images', collection_path)
```

Comment:
Should this path be hard-coded rather than a setting?

Reply:
Is it the case that we will only ever have one pending image copy operation enforced somewhere else in the code? If not, won't this be a problem if two operations are happening simultaneously and they're both copying files to the same hard-coded path?